docs/roadmap/STATUS.json

```
@@ -1,11 +1,7 @@
 {
-<<<<<<< HEAD
-  "last_updated": "2026-02-02T00:00:00Z",
-  "revision_note": "Normalized evidence/index.json to match evidence schema.",
-=======
+  "revision_note": "Added LongHorizon orchestration MVP scope and evidence references",
```

Biome (2.3.13)

[error] The key revision_note was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)